script/push-codecov.bash

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -x
+
+if [[ ${CI} = true ]]; then 
```

Comment:
```suggestion
if [[ "${CI}" = "true" ]]; then 
```

i think, i'm not positive `=` is the right operator but if it works its probably fine

Reply:
I think it should be `[ "$string1" = "string2" ]` or ` [[ "$string1" == "string2" ]]`